media-gfx/flameshot/flameshot-0.6.0.ebuild

```diff
@@ -24,6 +24,10 @@ DEPEND="
 	dev-qt/qtdbus:5
 	sys-apps/dbus
 "
+BDEPEND="
+	${DEPEND}
+	dev-qt/linguist-tools:5
+"
```

Comment:
`BDEPEND` is a split from `DEPEND`, so `BDEPEND="${DEPEND}"` is wrong by design :)

Simple `BDEPEND="dev-qt/linguist-tools:5"` works here. Rest looks good.

Reply:
Good now?